apps/studio/components/interfaces/DataWarehouse/CreateWarehouseCollection.tsx

```diff
@@ -103,6 +127,21 @@ export const CreateWarehouseCollectionModal = () => {
                   </FormItemLayout>
                 )}
               />
+              <FormField_Shadcn_
+                control={form.control}
+                name="retention_days"
+                render={({ field }) => (
+                  <FormItemLayout
+                    label="Retention days"
+                    layout="horizontal"
+                    description={retentionDescription()}
+                  >
+                    <FormControl_Shadcn_>
+                      <Input type="number" {...field} />
```

Comment:
let's follow how we put units in our input fields? ref settings/auth
and we can also remove "days" from the label too - so just "Retention"

![image](https://github.com/user-attachments/assets/48163b49-3d96-4cfb-810c-0f1b12c95f26)